tracing-attributes/src/lib.rs

```diff
@@ -176,6 +176,14 @@ mod expand;
 ///     fn my_method(&self) {}
 /// }
 /// ```
+/// Specifying `follows_from` relationships:
```

Comment:
could be worth a link to the `follows_from` method in the `tracing` documentation here?